### PR TITLE
Support v2.1 InstalledCode setup

### PR DESCRIPTION
**Pull request by unkcpz**
Tested at https://unkcpz.github.io/aiida-code-registry/

### CODE REVIEW

**Comment by unkcpz** [comment]
This one should be tested with awb before merge.

**Review by yakutovicha — CHANGES_REQUESTED** [review]
Thanks, @unkcpz, just one small request.

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ on:
 jobs:
 
     deploy-gh-pages:
-        if: github.repository == 'aiidateam/aiida-code-registry'
+        if: github.repository == 'unkcpz/aiida-code-registry'
```
[the remark this answers is not in the document]
Comment (yakutovicha) [reply .github/workflows/deploy.yml:13]: So you are not planning to merge it? If so, I guess it. makes sense to convert the PR to a draft.

[the remark this answers is not in the document]
Comment (unkcpz) [reply .github/workflows/deploy.yml:13]: Yep, changed.

**Review by yakutovicha — APPROVED** [review]

**Pull request merged**